task/github.py

```diff
@@ -142,6 +142,9 @@ def __init__(self, base=None, cacher=None, repo=None):
                 except FileNotFoundError:
                     pass
 
+        if self.token is None:
```

Comment:
I'm pretty sure there *are* things that can be done without a token.  `tests-scan` for example.  This would break that.

Reply:
Yeah, I just nuked the commit for now. A better approach would be to not return `None` when we receive a 404, this would require the callers to be updated and handle a possible `GithubStatus` error like in  https://github.com/cockpit-project/bots/blob/main/push-rewrite#L80